src/ui/components/Widgets/Chart/components/Content.tsx

```diff
@@ -102,6 +105,40 @@ export const Content = (props: ChartContentProps) => {
     const showContentLoader = showLoader || isExportLoading;
     const showLoaderVeil = veil && !isExportLoading;
 
+    const onAction = React.useCallback(async (actionArgs: {data?: any} = {}) => {
+        const {action, ...args} = actionArgs.data ?? {};
```

Comment:
I suggest to use `||` here. This code for example will crush execution
```js
// Possible args example (because of data: any)
const actionArgs = {data: false};
// ...
const {action, ...args} = actionArgs.data ?? {};
```